internal/task/task/common/sync_config.go

```diff
@@ -43,6 +43,16 @@ const (
 	CURVEFS_CRONTAB_FILE = "/tmp/curvefs_crontab"
 )
 
+// For tools.conf
+var S3INFODICT = map[string]string{
```

Comment:
What's the purpose?

Reply:
> What's the purpose?

The parameter format in curvefs_tool is -s3Sk. In order to maintain uniformity, the configuration file is also defined as s3Sk instead of s3.sk, so a map variable is used for conversion.